_docs-sources/intro/overview/intro-to-gruntwork.md

```diff
@@ -1,14 +1,12 @@
 
-All Gruntwork products exist within a [framework](/guides/production-framework) we’ve devised specifically to emphasize DevOps industry best-practices and maximize your team’s efficiency.
+Gruntwork works best for teams building new infrastructure (“greenfield”), either from scratch or as part of a migration. However, it can also be used by teams with existing infrastructure (“brownfield”) if they have sufficient DevOps experience. All Gruntwork products exist within a [framework](/guides/production-framework) we’ve devised specifically to emphasize DevOps industry best-practices and maximize your team’s efficiency.
 
-Gruntwork works best for teams building new infrastructure ("greenfield"), either from scratch or as part of a migration. However, it can also be used by teams with existing infrastructure ("brownfield") if they have sufficient DevOps experience.
-
-All Gruntwork products are built on and fully compatible with [Terraform](https://terraform.io). The one exception to this is the [Gruntwork Reference Architecture](https://gruntwork.io/reference-architecture/), which uses [Terragrunt](https://terragrunt.gruntwork.io/) (one of our open source tools) to implement an end-to-end architecture.
+All Gruntwork products are built on and fully compatible with [Terraform](https://terraform.io). The one exception to this is the [Gruntwork Reference Architecture](/refarch/whats-this/what-is-a-reference-architecture), which uses [Terragrunt](https://terragrunt.gruntwork.io/) (one of our open source tools) to implement an end-to-end architecture.
 
 There are two fundamental ways to engage Gruntwork:
 
-1. **Gruntwork builds your architecture.** We generate the [Reference Architecture](https://gruntwork.io/reference-architecture/) based on your needs, deploy into your AWS accounts, and give you 100% of the code. Since you have all the code, you can extend, enhance, and customize the environment exactly according to your needs. See [the docs](/refarch/whats-this/what-is-a-reference-architecture) for more information about our Reference Architecture.
-2. **Build it yourself.** The Gruntwork IaC library empowers you to construct your own bespoke architecture in record time. By mix-and-matching our [modules](/iac/overview/modules) and [services](/iac/overview/services) you can quickly define a custom architecture to suit your needs, all with the confidence of having world-class, battle-tested code running under the hood.
+1. **Gruntwork builds your architecture.** We generate the [Reference Architecture](/refarch/whats-this/what-is-a-reference-architecture) based on your needs, deploy into your AWS accounts, and give you 100% of the code. Since you have all the code, you can extend, enhance, and customize the environment exactly according to your needs. See [the docs](/refarch/whats-this/what-is-a-reference-architecture) for more information about our Reference Architecture.
```

Comment:
```suggestion
1. **Gruntwork builds your architecture.** We generate a Reference Architecture based on your needs, deploy into your AWS accounts, and give you 100% of the code. Since you have all the code, you can extend, enhance, and customize the environment exactly according to your needs. See [the docs](/refarch/whats-this/what-is-a-reference-architecture) for more information about our Reference Architecture.
```
Removes the duplicate link in the same "paragraph" to the Ref Arch docs